src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -397,8 +402,69 @@ def makeSubscriptions(self, workflow):
             pass
 
         elif self.msConfig['defaultDataManSys'] == 'Rucio':
-            pass
+            if isinstance(workflow, MSOutputTemplate):
+                msg = "Making transfer subscriptions for %s"
+                self.logger.info(msg, workflow['RequestName'])
 
+                rucioResultList = []
+                if workflow['destinationOutputMap']:
+                    for dMap in workflow['destinationOutputMap']:
+                        try:
+                            msg = "Making transfer subscriptions for %s"
+                            self.logger.info(msg, workflow['RequestName'])
+                            copies = workflow['numberOfCopies'] if workflow['numberOfCopies'] else 1
+                            # NOTE:
+                            #    Once we get rid of DDM this rseExpression generation
+                            #    should go in the Producer thread
+                            if workflow['isRelVal']:
+                                pnnExpr = '('+'|'.join(["pnn=%s" % pnn for pnn in dMap['destination']])+')'
+                                rseExpression = pnnExpr + '&cms_type=real&rse_type=DISK'
+                            else:
+                                rseExpression = '(tier=2|tier=1)&cms_type=real&rse_type=DISK'
+                            scope = 'cms'
```

Comment:
no need for this scope here, it has a default value defined in the method already.

Reply:
I know are never supposed to use anything else but the default 'cms' value comming from the method definition, which I cannot access from here, but I need it in order to build the porpper log message for the DRY_RUN mode, bellow. That's why I passed it explicitly here too, so I can reuse it later in the message.

Reply:
That's fine! If you want to show the scope in the DRY-RUN mode, this will have the same effect than writing `cms` in the log message though...

Reply:
Please update it as well.

Reply:
Done

Reply:
Done